openxr/src/generated.rs

```diff
@@ -3127,6 +3129,9 @@ pub mod raw {
     use sys::pfn;
     #[derive(Copy, Clone)]
     pub struct Instance {
+        pub negotiate_loader_runtime_interface: pfn::NegotiateLoaderRuntimeInterface,
+        pub negotiate_loader_api_layer_interface: pfn::NegotiateLoaderApiLayerInterface,
+        pub create_api_layer_instance: pfn::CreateApiLayerInstance,
```

Comment:
I don't think these are supposed to be application-facing. Should we be exposing them?

Reply:
I'm not sure how they compare to the rest of the function pointers. What are the trade-offs?

Reply:
Other function pointers are intended to be exposed by the loader and called by applications. These are, at a glance, intended to be exposed by runtimes/layers and called by the loader.

Reply:
Ok, I added a blacklist to remove these from the high level bindings.

Reply:
Is there nothing at all distinguishing them in the XML?

Reply:
Only a comment above that block of commands.

Reply:
```xml
    <!-- SECTION: OpenXR loader command definitions -->
    <!-- openxr_loader_negotiation.h -->
    <commands>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrNegotiateLoaderRuntimeInterface</name></proto>
            <param>const <type>XrNegotiateLoaderInfo</type>* <name>loaderInfo</name></param>
            <param><type>XrNegotiateRuntimeRequest</type>* <name>runtimeRequest</name></param>
        </command>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrNegotiateLoaderApiLayerInterface</name></proto>
            <param>const <type>XrNegotiateLoaderInfo</type>* <name>loaderInfo</name></param>
            <param len="null-terminated">const <type>char</type>* <name>layerName</name></param>
            <param><type>XrNegotiateApiLayerRequest</type>* <name>apiLayerRequest</name></param>
        </command>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrCreateApiLayerInstance</name></proto>
            <param>const <type>XrInstanceCreateInfo</type>* <name>info</name></param>
            <param>const <type>XrApiLayerCreateInfo</type>* <name>layerInfo</name></param>
            <param><type>XrInstance</type>* <name>instance</name></param>
        </command>
    </commands>
```

Reply:
That's annoying. Guess we'll just have to stay vigilant.